deps/fast_float/fast_float.h

```diff
@@ -1,3869 +0,0 @@
-// fast_float by Daniel Lemire
-// fast_float by João Paulo Magalhaes
-//
-//
-// with contributions from Eugene Golushkov
-// with contributions from Maksim Kita
-// with contributions from Marcin Wojdyr
-// with contributions from Neal Richardson
-// with contributions from Tim Paine
-// with contributions from Fabio Pellacini
-// with contributions from Lénárd Szolnoki
-// with contributions from Jan Pharago
-// with contributions from Maya Warrier
-// with contributions from Taha Khokhar
-//
-//
-// MIT License Notice
```

Comment:
Actually, I quite like this amalgamated version. It contains the license information too, which is important.

I believe they created this script just for projects like us, so that we can include only one file in our project.

Reply:
> I believe they created this script just for projects like us, so that we can include only one file in our project.
It makes it very difficult to compare or check for changes in future. I reverted it to single file though. If we are not going submodule route, I don't have preferences between single vs multi file :)